src/main/java/com/melon/portfoliomanager/services/KafkaService.java

```diff
@@ -0,0 +1,24 @@
+package com.melon.portfoliomanager.services;
+
+import org.apache.kafka.clients.admin.NewTopic;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.kafka.core.KafkaTemplate;
+import org.springframework.stereotype.Service;
+
+import java.util.Map;
+
+@Service("kafka")
+public class KafkaService implements MessageBrokerService {
+    private final KafkaTemplate<String, String> kafkaTemplate;
+    private final NewTopic topic;
+
+    @Autowired
+    public KafkaService(KafkaTemplate<String, String> kafkaTemplate, NewTopic topic) {
+        this.kafkaTemplate = kafkaTemplate;
+        this.topic = topic;
+    }
+
+    public void sendMessage(Map<String, ?> companyStocksWithExtremePriceChanges) {
```

Comment:
Change the param name? Should a KafkaService class be aware of companyStocksWithExtremePriceChanges, or just some message it needs to send?